changelog/1.25.0.md

```diff
@@ -3,6 +3,13 @@ title: "1.25.0"
 description: "Released on 11/17/2021"
 ---
 
+> The final patch release of Kubernetes 1.19 was published on 28 October 2021.
+> As such, Coder v1.25 (and later) require the use of Kubernetes v1.20+. See
```

Comment:
@khorne3 Ah, this should be the _next_ release (we haven't changed the requirement for 1.25) -- so should we change this to v1.26?

Reply:
Ah gotcha -- yeah, I think we should wait for the next release. I'm also spinning up an Upgrade guide, and I think this note is something that would belong on that doc too